README.md

```diff
@@ -126,7 +126,7 @@ Activate ESP-IDF PowerShell and enter following command:
 git clone https://github.com/esp-rs/rust-build.git
 cd rust-build
 ./Install-RustToolchain.ps1 -DefaultHost x86_64-pc-windows-gnu  -ExportFile Export-EspRust.ps1
-source Export-EspRust.ps1
+. Export-EspRust.ps1
```

Comment:
Just noticed and I am not sure if this works, doesn't it needs to be `. ./Export-EspRust.ps1`?

Reply:
Yes it does. Will fix.

Reply:
Just noticed you switched the base branch to `build/1.63.0.0` branch. I've updated the commit and rebased it onto that branch.